widget/playground/src/components/ItemPicker/ItemPicker.styles.ts

```diff
@@ -34,6 +34,9 @@ export const Container = styled('div', {
   display: 'flex',
   flexDirection: 'column',
   width: '100%',
+  '.title_typography:first-letter': {
```

Comment:
`textTransform` has a `capitalize` value as well. We don't need to select the first letter and make it uppercase.

```
 '.title_typography': {
    textTransform: 'capitalize',
  },
});
```

Reply:
fixed